app/stories/pix-input-password.stories.js

```diff
@@ -18,11 +18,6 @@ export default {
       description: "Label de l'input. Requis si ariaLabel n'est pas définit.",
       type: { name: 'string', required: true },
     },
-    ariaLabel: {
-      name: 'ariaLabel',
-      description: "L'action du champ, pour l'accessibilité. Requis si label n'est pas définit.",
-      type: { name: 'string', required: true },
-    },
     information: {
```

Comment:
l.60: dans le Template il y a encore un utilisation de l'ariaLabel `@ariaLabel={{this.ariaLabel}}`